### PR TITLE
ci: use conventional commit messages for generated PRs

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -58,8 +58,8 @@ jobs:
       if: github.event_name == 'push'
       uses: peter-evans/create-pull-request@v5
       with:
-        title: 'Regenerate mocks'
-        commit-message: 'Regenerate mocks'
+        title: 'chore: regenerate mocks'
```
Comment (CarlosNihelton) [inline .github/workflows/ci.yml:61]: I think this is more a matter of style than anything, but I like better `chore: Regenerate mocks.`, removing the CC prefix, it still looks like a sentence.

Comment (jpnurmi) [reply .github/workflows/ci.yml:61]: the [spec](https://www.conventionalcommits.org/en/v1.0.0/), dependabot, and renovate don't capitalize, though ;-)

- https://github.com/canonical/ubuntu-desktop-installer/pull/2017
- https://github.com/canonical/ubuntu-desktop-installer/pull/2021

Comment (CarlosNihelton) [reply .github/workflows/ci.yml:61]: I'm the weirdo then :D

**Review by CarlosNihelton — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jpnurmi) [reply .github/workflows/ci.yml:61]: Nah, I recall Daniel made the same comment at the engineering sprint. :) I wish GitHub had some nice label-like rendering for conventional commit prefixes...

**Pull request merged**